pkg/executor/wasm/executor_test.go

```diff
@@ -0,0 +1,71 @@
+package wasm
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/bacalhau-project/bacalhau/pkg/executor"
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+// Mocking necessary dependencies and structures
+
+type mockHandler struct {
+	mock.Mock
+}
+
+func (m *mockHandler) active() bool {
+	args := m.Called()
+	return args.Bool(0)
+}
+
+func (m *mockHandler) run(ctx context.Context) {
+	_ = m.Called(ctx)
+}
+
+type mockExecutor struct {
+	mock.Mock
+}
+
+func (m *mockExecutor) Get(executionID string) (*mockHandler, bool) {
+	args := m.Called(executionID)
+	return args.Get(0).(*mockHandler), args.Bool(1)
+}
+
+func (m *mockExecutor) Put(executionID string, handler *mockHandler) {
+	_ = m.Called(executionID, handler)
+}
```

Comment:
Is this used anywhere?

Reply:
ah forgot to remove it my bad @rossjones

Reply:
removed